istioctl/cmd/istioctl/main.go

```diff
@@ -97,6 +97,13 @@ var (
 		model.EgressRule.Type:        20,
 	}
 
+	// deprecatedTypes tracks if a deprecation warning is needed
+	deprecatedTypes = map[string]bool{
+		model.RouteRule.Type:         true,
+		model.IngressRule.Type:       true,
+		model.DestinationPolicy.Type: true,
```

Comment:
do we want EgressRule here as well? cc @vadimeisenbergibm

Reply:
Good idea; I added it.